west.yml

```diff
@@ -101,6 +101,10 @@ manifest:
       repo-path: sdk-mcuboot
       revision: v1.7.99-ncs4-rc2
       path: bootloader/mcuboot
+    - name: mbedtls-nrf
+      path: mbedtls
+      repo-path: mbedtls
```

Comment:
Should the repo path be: `sdk-mbedtls`

So that it follows the naming scheme used for `sdk-mcuboot`, `sdk-zephyr`, `sdk-trusted-firmware-m`, etc ?

Reply:
Done.

Thank you :)